includes/Dashboard/Templates/Orders.php

```diff
@@ -95,7 +95,7 @@ function handle_order_export() {
 
         $post_data = wp_unslash( $_POST );
 
-        if ( ! isset( $post_data['dokan_vendor_order_export_nonce'] ) || ! wp_verify_nonce( sanitize_key( $post_data['dokan_vendor_order_export_nonce'] ), 'dokan_vendor_order_export_action' ) ) {
+        if ( ! isset( $post_data['dokan_vendor_order_export_nonce'] ) || ! wp_verify_nonce( sanitize_key( $post_data['dokan_vendor_order_export_nonce'] ), 'dokan_vendor_order_export_action' ) || ! current_user_can( 'dokan_export_order' ) || ! current_user_can( 'administrator' ) || ! current_user_can( 'seller' ) ) {
```

Comment:
revert the changes as it was

Reply:
Here we are checking if the specific vendor staff has permission for exporting the order list as per our discussion yesterday.
Should we remove the check? Or maybe we could modify it with:
` if ( ! isset( $post_data['dokan_vendor_order_export_nonce'] ) || ! wp_verify_nonce( sanitize_key( $post_data['dokan_vendor_order_export_nonce'] ), 'dokan_vendor_order_export_action' ) || ! current_user_can( 'dokan_export_order' ) || ! current_user_can( 'manage_woocommerce' ) )`